cmd/admin-heal.go

```diff
@@ -96,20 +96,36 @@ var adminHealCmd = cli.Command{
 
 // healMessage container to hold repair information.
 type healMessage struct {
-	Status string             `json:"status"`
-	Bucket string             `json:"bucket"`
-	Object *madmin.ObjectInfo `json:"object"`
-	Upload *madmin.UploadInfo `json:"upload"`
+	Status string                  `json:"status"`
+	Result madmin.HealObjectResult `json:"result"`
```

Comment:
Do you mean madmin.HealResult here? (without object because it could be upload sometimes?)